include/iso15118/detail/cb_exi.hpp

```diff
@@ -23,6 +23,12 @@
     std::copy(in.begin(), in.end(), out.bytes);                                                                        \
     out.bytesLen = in.size()
 
+#define CB2CPP_BYTES(in, out)                                                                                          \
+    if (in.bytesLen > out.size()) {                                                                                    \
+        throw std::runtime_error("Byte array too long");                                                               \
+    }                                                                                                                  \
+    std::copy(std::begin(in.bytes), std::end(in.bytes), out.begin());
+
```

Comment:
Yes, that's right, but I wouldn't change that here in this PR. Because then we should also change all `#define` to templates. I'd rather do that in a separate PR.